.github/workflows/ci.yml

```diff
@@ -12,6 +12,14 @@ jobs:
       if: github.event_name == 'push'
     - uses: stepchowfun/toast/.github/actions/toast@master
       with:
-        tasks: build test lint run
+        tasks: build test lint run release-musl
         repo: stephanmisc/toast
         write_remote_cache: ${{ github.event_name == 'push' }}
+    - name: Integration Test
+      run: |
+        docker run --privileged --name dind -d \
+          -v $PWD/artifacts:/artifacts docker:dind
```

Comment:
Can we put the path with `$PWD` in double quotes? (Just in case the path has spaces in it for some reason.)